extensions/wikia/ImageReview/ImageListGetter.class.php

```diff
@@ -0,0 +1,312 @@
+	 * @return array
+	 */
+	private function getImageData( stdClass $imageRecord ) {
+		$imageInfo = $this->getBaseImageInfo( $imageRecord->wiki_id, $imageRecord->page_id );
+		if ( !empty( $imageInfo['error'] ) ) {
+			return $imageInfo;
+		}
+
+		$wikiRow = WikiFactory::getWikiByID( $imageRecord->wiki_id );
+		$cityUrl = !empty( $wikiRow ) && $wikiRow->city_url ? $wikiRow->city_url : '';
+
+		return [
+			'wikiId' => $imageRecord->wiki_id,
+			'pageId' => $imageRecord->page_id,
+			'state' => $imageRecord->state,
+			'src' => $imageInfo['src'],
+			'url' => $imageInfo['page'],
+			'extension' => $imageInfo['extension'],
+			'priority' => $imageRecord->priority,
+			'flags' => $imageRecord->flags,
+			'isthumb' => true,
+			'wiki_url' => $cityUrl,
+		];
+	}
+
+	private function getBaseImageInfo( $wikiId, $pageId ) {
+		$result = $this->getImageGlobalFile( $wikiId, $pageId );
+		if ( !empty( $result['error'] ) ) {
+			return $result;
+		}
+		/** @var GlobalFile $imageGlobalFile */
+		$imageGlobalFile = $result['file'];
+		/** @var GlobalTitle $imageGlobalPage */
+		$imageGlobalPage = $result['page'];
+
+		$thumbUrl = $imageGlobalFile->getUrlGenerator()
+			->width( self::IMAGE_REVIEW_THUMBNAIL_SIZE )
+			->height( self::IMAGE_REVIEW_THUMBNAIL_SIZE )
+			->thumbnailDown()
+			->url();
+
+		return [
+			'src' => $thumbUrl,
+			'page' => $imageGlobalPage->getFullURL(),
+			'extension' => $imageGlobalFile->getMimeType(),
+		];
+	}
+
+	/**
+	 * @param $pageId
+	 * @param $wikiId
+	 *
+	 * @return array
+	 *
+	 * @throws Exception
+	 */
+	private function getImageGlobalFile( $wikiId, $pageId ) {
+		try {
+			$imagePage = GlobalTitle::newFromId( $pageId, $wikiId );
+		} catch ( Exception $e ) {
+			return [ 'error' => 'Database for wiki does not exist' ];
+		}
+
+		if ( !$imagePage instanceof GlobalTitle ) {
+			return [ 'error' => 'Page does not exist' ];
+		}
+
+		if ( $imagePage->isRedirect() ) {
+			return [ 'error' => 'Page is a redirect' ];
+		}
+
+		$imageGlobalFile = new GlobalFile( $imagePage );
+		if ( !$imageGlobalFile->exists() ) {
+			return [ 'error' => 'File does not exist' ];
+		}
+
+		return [
+			'file' => $imageGlobalFile,
+		    'page' => $imagePage,
+		];
+	}
+
+	private function moveImagesToIcoStateFromWhere( array $where ) {
+		$count = count( $where );
+		if ( $count <= 0 ) {
+			return;
+		}
+
+		$values = [ 'state' => ImageReviewStates::ICO_IMAGE ];
+		$where = [ implode( 'OR', $where ) ];
+
+		$this->imageStateUpdater->updateBatchImages( $values, $where );
+
+		WikiaLogger::instance()->info( "ImageReviewLog", [
+			'method' => __METHOD__,
+			'message' => "Updated {$count} images (type 'icons')",
+		] );
+	}
+
+	private function getNewState() : int {
+		if ( $this->state == ImageReviewStates::REJECTED ) {
+			return ImageReviewStates::REJECTED_IN_REVIEW;
+		}
+
+		if ( $this->state == ImageReviewStates::QUESTIONABLE ) {
+			return ImageReviewStates::QUESTIONABLE_IN_REVIEW;
+		}
+
+		else return ImageReviewStates::IN_REVIEW;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)